test/tests/outputs/xml/tests

```diff
@@ -0,0 +1,28 @@
+[Tests]
+  issues = '#14634'
+  design = 'XMLOutput.md'
+  [xml]
+    requirement = "The system shall support XML output for vector data that is"
+
+    [replicated]
+      type = VTKDiff
+      input = xml.i
+      vtkdiff = xml_out.xml
+      min_parallel = 3
+      max_parallel = 3
+
+      detail = "replicated or "
```

Comment:
So do all of the "detail" strings get concatenated into one sentence?:
- `requirement` + " " + `detail1` + `detail2`

 I always thought it would be:
- `requirement` + " " + `detail1`
- `requirement` + " " + `detail2`

Reply:
Yes, they are one requirement with many parts. You can find some examples here:

https://mooseframework.org/sqa/framework_rtm.html